packages/server/src/util/version.ts

```diff
@@ -0,0 +1,13 @@
+import { readFileSync } from 'node:fs';
+import { resolve } from 'node:path';
+
+let serverVersion: string | undefined;
+
+export function getServerVersion(): string {
```

Comment:
We should not need this.  It should always be the same as `MEDPLUM_VERSION`

```ts
import { MEDPLUM_VERSION } from '@medplum/core';
```

Reply:
`MEDPLUM_VERSION` is not exactly the same as the version present in `package.json`. Also it makes it a lot harder to change the version for local testing purposes if you only refer to `MEDPLUM_VERSION` because it requires the version to be changed in `package.json` in `@medplum/core` and then build it in order to go back and forth between version numbers.

We can remove it but that means everywhere we have to in-line process `MEDPLUM_VERSION` to account for the fact that it could have a hash or maybe not

Reply:
Would it be ok to add `getServerVersion` which strips the hash from the version?